docs/kafka/metrics-monitoring-kafka/README.adoc

```diff
@@ -87,76 +87,39 @@ As a developer or administrator, you can view metrics in {product-kafka} to visu
 == Supported metrics in {product-kafka}
 
 [role="_abstract"]
-{product-kafka} supports the following metrics for Kafka instances and topics. In the {product-kafka} web console, a subset of these metrics is displayed in the *Dashboard* page of each Kafka instance.
+{product-kafka} supports the following metrics for Kafka instances and topics. In the {product-kafka} web console, a subset of these metrics is displayed in the *Dashboard* page of each Kafka instance. To learn more about the limits associated with both trial and production Kafka instances, refer to https://access.redhat.com/articles/5979061[Red Hat OpenShift Streams for Apache Kafka Service Limits].
```

Comment:
Can eliminate passive voice here:

```
In the {product-kafka} web console, the *Dashboard* page of a Kafka instance displays a subset of these metrics.
```

Reply:
I think I'd make a connection here to the phrase "Kafka instance type" that we use extensively below.

"To learn about the limits associated with both trial and production Kafka instance types..."